aries-site/src/layouts/content/Example/BrowserWrapper.js

```diff
@@ -22,7 +22,7 @@ export const BrowserWrapper = ({ screen, ...rest }) => {
           direction="row"
           flex={false}
           gap="xsmall"
-          background="background-contrast"
+          background={{ color: 'border-weak' }}
```

Comment:
This seems odd to be using a "border" color for a background. Can we add a comment here explaining why?
```suggestion
          /* This Box represents a header of a browser application window. The selected background color 
           provides better contrast between the header and the examples being showcased. */
          background={{ color: 'border-weak' }}
```

Reply:
Yes, sounds good.